documentation/schemas/user.js

```diff
@@ -3,36 +3,54 @@ module.exports = {
     type: 'integer',
     example: 7
   },
-  username: {
+  firstName: {
     type: 'string',
-    example: 'tom99'
+    example: 'rick'
+  },
+  lastName: {
+    type: 'string',
+    example: 'sanchez'
   },
   userEmail: {
     type: 'string',
-    example: 'tom.engels@wolox.com.ar'
+    example: 'rick.sanchez@wolox.com.ar'
   },
-  User: {
+  password: {
+    type: 'password',
+    example: '14_fsda_21'
```

Comment:
Just a comment. If the pass allows symbols and uppercases, it's good to add them to the example